app/views/finance/ecf/statements/show.html.erb

```diff
@@ -142,6 +142,143 @@
     </div>
   </div>
 </div>
+<% if @statement.cohort.mentor_funding? %>
```

Comment:
same here: we have a ticket for the FE finance calculators, can we please leave out the changes from this one?